src/Symfony/Component/HttpKernel/Profiler/PdoProfilerStorage.php

```diff
@@ -11,11 +11,16 @@
 
 namespace Symfony\Component\HttpKernel\Profiler;
 
+@trigger_error('The '.__NAMESPACE__.'\PdoProfilerStorage class is deprecated since Symfony 2.8 and will be removed in 3.0. Use FileProfilerStorage instead.', E_USER_DEPRECATED);
```

Comment:
this should also be triggered for child classes IMO, as they are also deprecated.